### PR TITLE
Katerina classes ru

### PR DESCRIPTION
**Pull request by AliaksandrSumich**
Сделали статью. Можно проверять

### CODE REVIEW

File: ru/learn/classes.md [hunk 1]
```diff
@@ -1,4 +1,377 @@
...
+        print(f"Hello, my name is {self.name} and I am {self.age} years old.")
+
+person = Person("John", 30)
+person.greet()
+
+# Output:
+# 
+# Hello, my name is John and I am 30 years old.
+```
+В этом примере класс `Person` имеет два атрибута: `name` и `age`, а также метод `greet`, который выводит приветственное сообщение. Метод `__init__` является конструктором, который инициализирует атрибуты объекта. Чтобы создать объект класса `Person`, мы используем конструктор класса и передаем в него необходимые параметры.
```
Comment (angarsky) [inline ru/learn/classes.md:42]: Empty line is required here.

File: ru/learn/classes.md [hunk 2]
```diff
@@ -1,4 +1,377 @@
...
+        if amount > self.balance:
+            raise ValueError("Not enough funds")
+        self.balance -= amount
+        self.transactions.append(("withdrawal", amount))
+
+my_account = BankAccount("123456", 500)
+my_account.deposit(200)
+my_account.withdraw(50)
+```
+В этом примере класс `BankAccount` определен с методом `__init__`, который устанавливает атрибуты `account_number` и `balance`. Также инициализируется список `transactions`, который будет отслеживать все транзакции по счету. Методы `deposit` и `withdraw` могут быть вызваны для объекта `BankAccount` для изменения баланса счета и добавления транзакции. Наконец, переменная `my_account` создается как объект `BankAccount` с начальным балансом `500`,  счет модифицируется с помощью `deposit` и снятия средств.
```
Comment (angarsky) [inline ru/learn/classes.md:154]: Empty line is required.

File: ru/learn/classes.md [hunk 3]
```diff
@@ -1,4 +1,377 @@
...
+
+Затем мы можем получить доступ к атрибутам и методам объекта, используя точечную нотацию.
+
+В итоге, классы в Python позволяют нам создавать объекты с атрибутами и методами, а экземпляры класса создаются путем вызова класса, как функции.
+
+## Методы и функции в Python
+
+Методы и функции в Python — это два важных программных концепта. Метод — это функция, которая ассоциирована с объектом, в то время как функция не ассоциирована ни с каким объектом. Другими словами, метод — это функция, которая принадлежит классу, в то время как функция не связана ни с каким классом.
+
+В Python метод обычно вызывается в форме `object.method()`, в то время как функция вызывается в форме `function()`. Методы используются для выполнения действий над объектом или для `возвращения` значения, связанного с объектом. С другой стороны, функции используются для выполнения задачи или операции над данными и `возвращения` результата.
```
Comment (angarsky) [inline ru/learn/classes.md:196]: \`возвращения\` => \`return\`

File: ru/learn/classes.md [hunk 4]
```diff
@@ -1,4 +1,377 @@
...
+# Invoking class method without creating an instance
+
+MyClass.greet()
+
+# Creating an instance and invoking instance method
+
+obj = MyClass("Alice")
+obj.say_hello()
+```
+В этом примере у нас есть класс под названием `MyClass` с двумя методами: `greet()` и `say_hello()`.
```
Comment (angarsky) [inline ru/learn/classes.md:261]: Empty line is required here.

**Review by angarsky — CHANGES_REQUESTED** [review]
Good start! But there are some mistakes: 3 missed empty lines after code samples + 1 translated technical word.

**Comment by angarsky** [comment]
About empty lines.

**Comment by AliaksandrSumich** [comment]
Fixed

**Pull request merged**